ConsoleView.cpp

```diff
@@ -1,26 +1,41 @@
+//All these functions return void, they are simply here 
+//for the user to call when they want to get information on 
+//a CLASS (or a list of all classes), RELATIONSHIP, or ATTRIBUTE
 
-int UserInputLoop()
-{
-    std::vector<UMLClass> Classes;
-    int count = 0;
-    std::cout << "Enter the amount of Classes to add:" << std::endl;
-    std::cin >> count;
+//Prtint class will be used to print the name of a class given a UMLClass object
+//By using the get_class_name function from the UMLClass, it then prints the name of the Class given
+//@Para: UMLClass OBJ
+PrintClassName(UMLClass X){
```

Comment:
- All of these are missing the return type as well as the namespace. So I don't think this would compile? They all need to have "void" before the method name.
- They're also missing the namespace qualifier in front of the method name, which would definitely not be able to compile. 
- Our standard for method names have been all lower case separated by underscores. I'd like to keep it consistent across the project.

All those considered, the header for this and any other method in this file would look something like `void ConsoleView::print_class_name(UMLClass X)`

Reply:
Thank you I forgot this was a .cpp file